app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,402 @@
+        <item>1.5x</item>
+        <item>2x</item>
+        <item>3x</item>
+        <item>4x</item>
+        <item>8x</item>
+    </string-array>
+
+    <string-array name="rom_icon_filtering_options">
+        <item>Ninguno (pixelado)</item>
+        <item>Lineal (suave)</item>
+    </string-array>
+
+    <string-array name="game_runtime_console_type_options">
+        <item>Predeterminado</item>
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="firmware_settings_language_options">
+        <item>Inglés</item>
+        <item>Francés</item>
+        <item>Alemán</item>
+        <item>Italiano</item>
+        <item>Español</item>
+        <item>Japonés</item>
+    </string-array>
+
+    <string-array name="game_runtime_mic_source_options">
+        <item>Predeterminado</item>
+        <item>Ninguno</item>
+        <item>Soplar</item>
+        <item>Micrófono del dispositivo</item>
+    </string-array>
+
+    <string-array name="console_type_options">
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="fps_counter_position_options">
+        <item>escondido</item>
+        <item>Arriba-izquierda</item>
+        <item>Centro-superior</item>
+        <item>Arriba-derecha</item>
+        <item>Abajo-izquierda</item>
+        <item>inferior-central</item>
+        <item>Abajo-derecha</item>
+    </string-array>
+
+    <string-array name="video_filtering_options">
+        <item>ninguno</item>
+        <item>lineal</item>
+        <item>2xBR</item>
+        <item>HQ2X</item>
+        <item>HQ4X</item>
+        <item>Quilez</item>
+        <item>LCD</item>
+    </string-array>
+
+    <string-array name="audio_interpolation_options">
+        <item>ninguno</item>
+        <item>lineal</item>
+        <item>coseno</item>
+        <item>cúbico</item>
+    </string-array>
+
+    <string-array name="audio_bitrate_options">
+        <item>Automático</item>
+        <item>10-bits</item>
+        <item>16-bits</item>
+    </string-array>
+
+    <string-array name="audio_latency_options">
+        <item>Bajo (más lento)</item>
+        <item>Medio</item>
+        <item>Alto (más rápido)</item>
+    </string-array>
+
+    <string-array name="mic_source_options">
+        <item>ninguno</item>
+        <item>soplar</item>
+        <item>Micrófono del dispositivo</item>
+    </string-array>
+
+    <string-array name="save_state_locations">
+        <item>Guardar directorio de archivos</item>
+        <item>Directorio de ROMs</item>
+        <item>Directorio interno</item>
+    </string-array>
+
+    <string-array name="layout_orientation_options">
+        <item>Seguir sistema</item>
+        <item>retrato</item>
+        <item>paisaje</item>
+    </string-array>
+
+    <string-array name="background_portrait_mode_options">
+        <item>elasticidad</item>
+        <item>Centro de ajuste</item>
```

Comment:
This translation has a name but the original text has a verb. What do you think about using "Ajuste al centro" instead? The same could apply to the same string in the `background_landscape_mode_options` array below.